tools/super-tox.py

```diff
@@ -476,50 +497,66 @@ def main(
         handlers=[rich.logging.RichHandler()],
     )
 
-    if mode in ("local", "lxd-per-tox"):
+    assert mode in ("local", "lxd", "lxd-per-tox")
+    settings.mode = mode
+    settings.lxd_name = lxd_name
+    settings.lxd_image_alias = lxd_image_alias
+    settings.keep_lxd_instance = keep_lxd_instance
+    settings.cache_folder = pathlib.Path(cache_folder)
+    settings.ops_source = ops_source
+    settings.ops_source_branch = ops_source_branch
+    settings.remove_local_changes = remove_local_changes
+    settings.git_pull = git_pull
+    settings.repo_re = repo
+    settings.fresh_tox = fresh_tox
+    settings.workers = workers
+    settings.verbose = verbose
+    settings.sample = sample
+    del (
+        mode,
+        lxd_name,
+        lxd_image_alias,
+        keep_lxd_instance,
+        cache_folder,
+        ops_source,
+        ops_source_branch,
+        remove_local_changes,
+        git_pull,
+        repo,
+        fresh_tox,
+        workers,
+        verbose,
+        sample,
+    )
```

Comment:
There doesn't seem to be much point in this? These are all pretty small objects so refcounting them to zero here won't save much.

If we do want to save the memory then I think I would rather move everything below to a separate function that gets passed the settings and let these go out of scope normally.

Reply:
Let me see if I can do this somehow differently, maybe click.group() and/or click.pass_context.